### PR TITLE
Migrating config to autodetection

### PR DESCRIPTION
**Pull request by maxzieba**
## Proposed changes

Please explain the changes you made. Add pictures or videos to explain them better if appropriate.

## Checklist

Before posting a PR ensure that from each of the below categories **AT LEAST ONE BOX HAS BEEN CHECKED**. If more than one category is applicable then more can be checked. Also ensure that the proposed changes have been filled out with relevant information for reviewers.

## Tests

- [ ] No tests required to be added. (For small changes that will be tested by CI/CD infrastructure).
- [ ] Added/Modified automated and PhantomHand CI tests (if a new class is added (Python or C++), the interface of that class must be unit tested).
- [ ] Manually tested in simulation (if simulation specific or no hardware required to test the functionality). 
- [ ] Manually tested on hardware (if hardware specific or related).

## Documentation

- [ ] No documentation required to be added.
- [ ] Added documentation (For any new feature, explain what it does and how to use it. Write the documentation in a relevant space, e.g. Github, Confluence, etc).
- [ ] Updated documentation (For changes to pre-existing features mentioned in the documentation).

### CODE REVIEW

File: sr_hand_config/4650/controls/host/common/friction_compensation.yaml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+sr_friction_map: [
+#["LFJ5", 1,  [[0.0, 0.0], [1.0, 0.0]]],
```
Comment (bdenoun) [inline sr_hand_config/4650/controls/host/common/friction_compensation.yaml:2]: IS there any reason to keep this commented out?

Comment (maxzieba) [reply sr_hand_config/4650/controls/host/common/friction_compensation.yaml:2]: That's how we have it across most if not all hands.

Comment (bdenoun) [reply sr_hand_config/4650/controls/host/common/friction_compensation.yaml:2]: And? Is there a reason why? Keeping commented out code is definitely not a good idea

Comment (maxzieba) [reply sr_hand_config/4650/controls/host/common/friction_compensation.yaml:2]: I don't think I am the right person to ask that. I have a few questions myself regarding the controller files (this config is for a hand lite yet, we have controllers defined for LF which I don't understand why). I am having a chat with Toni later, looking for some explanation and I am happy to add your question to the list. Might be worth having a look again at the structure and contents of the config files :)

**Review by beatrizleon — APPROVED** [review]

**Pull request merged**